.travis.yml

```diff
@@ -15,8 +19,8 @@ cache:
     - ci/cache
 deploy:
   provider: script
-  script: bash -o xtrace ci/deploy.sh
+  script: bash ci/deploy.sh
   skip_cleanup: true
   on:
     branch: master
-    condition: $TRAVIS_EVENT_TYPE = "push"
+    condition: $TRAVIS_EVENT_TYPE = "push" && -v MANUBOT_SSH_PRIVATE_KEY
```

Comment:
Good idea. This way we can continue supporting the encrypted file travis deploy keys.

Reply:
Wait actually this breaks the deploy for travis users still using the old method, because `MANUBOT_SSH_PRIVATE_KEY` is unset for them. Hmm not sure what to do.

Reply:
Oh, right.  I had the logic mixed up.